sponsor-dapp-v2/src/components/Step1.js

```diff
@@ -42,7 +42,7 @@ function Step1(props) {
   });
 
   return (
-    <div className="step step--primary">
+    <>
```

Comment:
Why this rather than `<div>`?

Reply:
I think this "fragments" feature [1] is the recommended (or at least the new and shiny) way to return multiple children, rather than a wrapper div. Is there a reason why we'd prefer the div?

1. https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
SGTM - this definitely mirrors the original DOM more closely than adding a new div.